src/database/initialize.ts

```diff
@@ -9,7 +11,11 @@ export const initialize = async () => {
         await TaskModel.deleteMany();
         console.log("Creating dummy tasks");
         await TaskModel.create(tasksJson);
-    } catch(error) {
+        console.log("Dropping users table!");
+        await UserModel.deleteMany();
+        console.log("Creating users");
+        await UserModel.create(usersJson);
```

Comment:
I assume it generates the id here for users and tasks, is that why you set _id to null?

Reply:
Yes, not passing an _id, generates a unique one